File: loianeTraining/javaBasico/exercicios-aulas-18-19/Ex27.java
public class Ex27 {
    public static void main(String[] args) {
        int[] a = { 1, 2, 4, 7, 8, 10, 16, 8, 9, 10 };
        char[] b = new char[10];

        for (int i = 0; i < b.length; i++) {
            if (a[i] < 7) {
                b[i] = 'a';
            } else if (a[i] == 7) {
                b[i] = 'b';
            } else if (a[i] > 7 && a[i] < 10) {
                b[i] = 'c';
            } else if (a[i] == 10) {
                b[i] = 'd';
            } else {
                b[i] = 'e';
            }
            System.out.println(b[i]);
        }
    }
}

File: loiane-training-csharp/procedural-poo/d/Ex27.cs
using System;

class Ex27
{
    static void Main(string[] args)
    {
        Console.WriteLine("Quantas turmas existem na escola? ");
        int quantidadeDeTurmas = int.Parse(Console.ReadLine());

        int totalAlunos = 0;
        for (int i = 0; i < quantidadeDeTurmas; i++)
        {
            int quantidadeDeAlunos;
            do
            {
                Console.WriteLine("Quantos alunos existem na turma {0}? ", i + 1);
                quantidadeDeAlunos = int.Parse(Console.ReadLine());
            } while (quantidadeDeAlunos < 0 || quantidadeDeAlunos > 40);

            totalAlunos += quantidadeDeAlunos;
        }

        double media = totalAlunos / quantidadeDeTurmas;

        Console.WriteLine("Media de alunos nas turmas e {0}", media);
    }
}